### PR TITLE
FIX: DOM operations should be in afterRender not next

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: assets/javascripts/discourse/components/full-page-chat.js [hunk 1]
```diff
@@ -57,12 +60,14 @@ export default Component.extend({
...
-      sidebarScroll.scrollTop = sidebarScroll.scrollHeight;
-    }
+    schedule("afterRender", () => {
+      const sidebarScroll = document.querySelector(
+        ".sidebar-container .scroll-wrapper"
+      );
+      if (sidebarScroll) {
+        sidebarScroll.scrollTop = sidebarScroll.scrollHeight;
+      }
+    });
```
Comment (CvX) [inline assets/javascripts/discourse/components/full-page-chat.js:70]: This one is called from `didInsertElement`, so maybe there's no need to delay it?

Comment (jjaffeux) [reply assets/javascripts/discourse/components/full-page-chat.js:70]: yep I'm actually doing few tests on this, agreed it's probably overkill

Comment (jjaffeux) [reply assets/javascripts/discourse/components/full-page-chat.js]: @CvX look at last changes please, I think that should actually be enough

**Review by CvX — APPROVED** [review]
Looks good! The PR title and commit message need to be updated though 😃

**Pull request merged**